content/docs/overview.md

```diff
@@ -1242,6 +1242,14 @@ ok := key in m; // `ok` is true if the element for that key exists
 
 The first approach is called the "comma ok idiom".
 
+You can also initialize maps with map literals:
+```odin
+m:map[string]int = {
```

Comment:
I would write this as: 
```odin
m := map[string]int{
```
to make it clearer

Reply:
Makes sense. Fixed!